pandas/tests/io/test_html.py

```diff
@@ -3,13 +3,17 @@
 import glob
 import os
 import re
+import threading
 import warnings
 
+
+# imports needed for Python 3.x but will fail under Python 2.x
```

Comment:
if this is not supported in py2, then you should either conditionally import it, or set ``reload=None`` and check it (and skip in a test).